vmware_aria_operations_integration_sdk/mp_test.py

```diff
@@ -702,6 +717,22 @@ def main() -> None:
         help="Simulate the 'collect' method being called by the VMware Aria Operations collector.",
     )
     collect_method.set_defaults(func=run_collect)
+    collect_method.add_argument(
+        "-n",
+        "--collection-number",
+        help="Start at a custom collection number instead of 0.",
+        type=int,
+        default=0,
+        choices=range(0, 1000),
```

Comment:
We can't have an open-ended range while also excluding negative numbers. I think this is a fair compromise.